### ISSUE TITLE
Heater cooler service vs thermostat

### ISSUE THREAD

**Issue by leehadassin** [mihailescu2m/homebridge-airtouch4-platform#13]
Thanks for making this.
I have developed a plug-in for the airtouch2+ system using your plug-in as a starting point. I plan to fork officially and publish my version hopefully soon.

Just wondering if you have looked at using a heater cooler service type rather than a thermostat. Benefit is the heater cooler can integrate the fan speed in the native home.app.

Thanks again for your awesome plug-in.
Appreciate your hard work!

Regards

Lee

**Comment by mihailescu2m** [comment]
Yes I did, however after implementing both ways, heater cooler had some usability issues...
